idea/src/org/jetbrains/kotlin/idea/actions/KotlinAddImportAction.kt

```diff
@@ -155,7 +161,31 @@ class KotlinAddImportAction internal constructor(
             return true
         }
 
-        JBPopupFactory.getInstance().createListPopup(getVariantSelectionPopup()).showInBestPositionFor(editor)
+        val popup = object : ListPopupImpl(getVariantSelectionPopup()) {
+            @Suppress("UNCHECKED_CAST")
+            override fun getListElementRenderer(): ListCellRenderer<AutoImportVariant> {
+                val baseRenderer = super.getListElementRenderer() as PopupListElementRenderer<AutoImportVariant>
+                val psiRenderer = DefaultPsiElementCellRenderer()
+                return ListCellRenderer { list, value, index, isSelected, cellHasFocus ->
+                    JPanel(BorderLayout()).apply {
+                        baseRenderer.getListCellRendererComponent(list, value, index, isSelected, cellHasFocus)
+                        add(baseRenderer.nextStepLabel, BorderLayout.EAST)
+                        add(
+                            psiRenderer.getListCellRendererComponent(
+                                list,
+                                value.declarationToImport(project),
+                                index,
+                                isSelected,
+                                cellHasFocus
+                            )
+                        )
+                    }
+                }
+            }
+        }
+        NavigationUtil.hidePopupIfDumbModeStarts(popup, project)
```

Comment:
Present in java action but was missing here. Is it still required?

Reply:
No, it is unnecessary.

Reply:
fixed